download_traffic_data.py

```
@@ -0,0 +1,234 @@
+                country="JP"
+            )]
+        
+        # Prepare output file
+        timestamp = datetime.now().strftime('%Y%m%d_%H%M%S')
+        filename = f"jp_traffic_{timestamp}.csv"
+        output_path = storage.get_processed_dir('jartic') / filename
+        output_path.parent.mkdir(parents=True, exist_ok=True)
+        
+        # Download data
+        total_measurements = 0
+        with open(output_path, 'w') as f:
```

Ruff (0.12.2)

Async functions should not open files with blocking methods like `open`

(ASYNC230)